trade-finance-manager-ui/styles/table.scss

```diff
@@ -160,6 +160,7 @@ table.premium-payments-table {
 
 table.utilisation-report-reconciliation-table {
   table-layout: fixed;
+  width: calc(100% - 1px);
```

Comment:
Please could we add a comment in the code here with a brief explanation as to why?

Reply:
Apologies, added a block comment to provide reasoning for this